config/locales/en.yml

```diff
@@ -376,13 +376,17 @@ en:
         update_failure: "Could not save changes. Error: %{error_msg}"
         delete: "Delete order type"
         confirm_delete: "There are %{count} listings with this order type. If you delete the order type, those listings will be closed. Are you sure you want to delete the order type?"
+        confirm_delete_singular: "There is %{count} listing with this order type. If you delete the order type, this listings will be closed. Are you sure you want to delete the order type?"
```

Comment:
Plurals are wrong. Either listing or listings - probably listing in both places.